### PR TITLE
Dying specific Items

### PR DESCRIPTION
**Pull request by Jeryn99**
This PR allows the player to change the colour of a item

### CODE REVIEW

File: common/src/main/java/whocraft/tardis_refined/client/TRItemColouring.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+public class TRItemColouring {
+
+    public static ItemColor SCREWDRIVER_COLORS = (itemStack, tintIndex) -> {
+        if (tintIndex == 0) {
+            if (itemStack.getItem() instanceof DyeableLeatherItem dyeableLeatherItem) {
+                return dyeableLeatherItem.getColor(itemStack);
+            }
+        }
+        return -1;
```
Comment (50ap5ud5) [inline common/src/main/java/whocraft/tardis_refined/client/TRItemColouring.java]: What does this magic number mean?

File: common/src/main/java/whocraft/tardis_refined/client/TRItemColouring.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package whocraft.tardis_refined.client;
+
+import net.minecraft.client.color.item.ItemColor;
+import net.minecraft.world.item.DyeableLeatherItem;
+
+public class TRItemColouring {
+
+    public static ItemColor SCREWDRIVER_COLORS = (itemStack, tintIndex) -> {
```
Comment (50ap5ud5) [inline common/src/main/java/whocraft/tardis_refined/client/TRItemColouring.java:9]: What colour does this refer to? The Emitter? Needs some docs pls

**Review by 50ap5ud5 — CHANGES_REQUESTED** [review]
Looks great, some docs would be nice. :)

File: common/src/main/java/whocraft/tardis_refined/client/renderer/blockentity/device/AstralManipulatorRenderer.java [hunk 3]
```diff
@@ -83,7 +88,14 @@ public void render(AstralManipulatorBlockEntity blockEntity, float partialTick,
             poseStack.pushPose();
             poseStack.translate(-posAOffsetX, -posAOffsetY, -posAOffsetZ);
             VertexConsumer vertexBuilder = bufferSource.getBuffer(RenderType.lightning());
-            RenderHelper.drawGlowingBox(poseStack, vertexBuilder, length + 1.25f,  height + 1.25f , width + 1.25f,  0.635f, 0.392f, 0.878f,  0 + sine , 0 );
+          //  RenderHelper.drawGlowingBox(poseStack, vertexBuilder, length + 1.25f,  height + 1.25f , width + 1.25f,  0.635f, 0.392f, 0.878f,  0 + sine , 0 );
```
Comment (CommandrMoose) [inline common/src/main/java/whocraft/tardis_refined/client/renderer/blockentity/device/AstralManipulatorRenderer.java]: Commented out line.

**Review by CommandrMoose — CHANGES_REQUESTED** [review]

File: gradle.properties [hunk 4]
```diff
@@ -49,7 +49,7 @@ neoforge_data_pack_format=18
 mod_version=1.0.1
 license=https://github.com/CommandrMoose/TardisRefined/blob/master/LICENSE
 credits=Commoble, Lucraft, starray1000000
-mod_authors=CommandrMoose, Suff99, Magic Man, ILikePandas, Jacob K, 50ap5ud5
+mod_authors=CommandrMoose, Jeryn99, Magic Man, ILikePandas, Jacob K, 50ap5ud5
 mod_description=TARDIS: Refined is a Minecraft mod based on the world of Doctor Who that adds a personal, teleporting base to the game. The mod explores the nature of the TARDIS and how it operates and expresses its personality.
 display_url=https://github.com/CommandrMoose/TardisRefined
```
Comment (CommandrMoose) [inline gradle.properties]: Might want to update it here too whilst you're at it.

**Review by CommandrMoose — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 4 above)
Comment (Jeryn99) [reply gradle.properties]: Consider it done, same goes for Modrinth and Curse

**Review by CommandrMoose — APPROVED** [review]

**Pull request merged**